src/registrar/admin.py

```diff
@@ -742,9 +743,6 @@ def overridden_email_field(self, obj):
                 "fields": (
                     "is_active",
                     "groups",
-                    "portfolio",
```

Comment:
All of the admin changes work great, but thinking about it from an analyst perspective it seems inconvenient to no longer be able to view/edit anything about portfolios from within the user admin page. With these changes, if I want to see all portfolios a user is a part of the best I can do (far as I can tell at least) is go to the user_portfolio_permissions page, sort by user, and ctrl+F for the user I want. That's not the worst thing ever, but is there any easy way to make this more accessible?

Reply:
cc: @abroddrick I think this would need to be another ticket as it would increase the scope of this ticket quite a bit.  I think it is a good comment though worth considering.
We just did another ticket which lists the users in the portfolio page.  This would essentially be the opposite, listing the portfolios in the user page, right?

Reply:
Yes agreed that 100 percent would need to be it's own ticket. On a related not, when looking at this message I noticed there wasn't a direct relationship between user and the portfolio permission table, such as "portfolios-(models.ManyToManyField("registrar.Portfolio",through="registrar.PortfolioPermission",
        related_name="portfolio_users",)" Which would allow us to get the portfolios for a user with just doing user.portfolios as opposed to any other filtering (and is slightly faster django wise). Was there a reason this was excluded. (I also looked very fast so forgive me if I missed it or something similar to it.

Reply:
So time wise, the conversion to values list is slower for getting portfolios vs having a relation. Funny you point out UserDomainRoles though as that one is also connected on User with "domains". This was done for the same reason to allow for fast queries to get the relation and avoid the code of having to query the table and then get the series fitting "domains".
```
domains = models.ManyToManyField(
        "registrar.Domain",
        through="registrar.UserDomainRole",
        related_name="users",
    )
```

Reply:
Okay side note though, I just saw that you have portfolio_permissions as the related name, but then also have a function with the same name and another function called get_portfolio_permissions. This made tracing the code rather rough, if I didn't know to look for one that was a relation, it would have taken a lot longer, I would highly encourage renaming the relation or renaming the functions so someone who is new to the codebase isn't confused on which of the multiple portfolio_permissions is being referred to.

Reply:
Also to be clear I am not blocking on any of this, just asking nosey questions.

Reply:
[ticket](https://github.com/cisagov/manage.get.gov/issues/2671) for adding portfolio back on the User. Keeping it simple for the first go, we can iterate on it after some more feedback from analysts